calypso/csadmin/clicontracts/lts_test.sh

```diff
@@ -0,0 +1,50 @@
+# This method should be called from the calypso/csadmin/test.sh script
+
+testContractLTS() {
+    run testContractLTSInvoke
+}
+
+testContractLTSInvoke() {
+    # -bc not given
+    testFail runCA contract lts spawn
+
+    rm -f config/*
+    runCoBG 1 2 3
+    runGrepSed "export BC=" "" runBA create --roster public.toml --interval .5s
+    eval $SED
+    [ -z "$BC" ] && exit 1
+
+    OUTRES=`runCA contract lts spawn`
+    matchOK "$OUTRES" "^Spawned a new LTS contract. Its instance id is:
+[0-9a-f]{64}$"
+
+    # Create a DARC
+    testOK runBA darc add -out_id ./darc_id.txt -out_key ./darc_key.txt -unrestricted
+    ID=`cat ./darc_id.txt`
+    KEY=`cat ./darc_key.txt`
+
+    # Fail because it does not have the "spawn:longTermSecret"
+    testFail runCA contract lts spawn --darc "$ID"
+
+    # Fail because this identity is not allowed in the default used admin darc
+    #
+    # This test should be done, but we end up with a "Got a duplicate
+    # transaction, ignoring it" later it we do. 
```

Comment:
could you prepend the comment with a TODO? so it's easy to grep these kind of things later